src/main/java/common/tileentities/GTMTE_LapotronicSuperCapacitor.java

```diff
@@ -1149,7 +1157,24 @@ public void addUIWidgets(ModularWindow.Builder builder, UIBuildContext buildCont
             .addTooltip(StatCollector.translateToLocal("gui.kekztech_lapotronicenergyunit.wireless"))
             .setTooltipShowUpDelay(TOOLTIP_DELAY))
             .widget(new FakeSyncWidget.BooleanSyncer(() -> wireless_mode, val -> wireless_mode = val))
-            .widget(new FakeSyncWidget.BooleanSyncer(this::canUseWireless, val -> canUseWireless = val));
+            .widget(new FakeSyncWidget.BooleanSyncer(this::canUseWireless, val -> canUseWireless = val))
+            .widget(new ButtonWidget().setOnClick((clickData, widget) -> {
+                if (wireless_mode && canUseWireless && !balanced) {
```

Comment:
```suggestion
                if (mMachine && wireless_mode && canUseWireless && !balanced) {
```
Probably don't want to allow rebalancing unformed LSCs.